rules/enforce-placeholder-last.js

```diff
@@ -44,9 +46,19 @@ module.exports = {
 				return;
 			}
 
+			const placeholder = topLevelStatements[firstPlaceholderIndex];
+			const text = sourceCode.getText(placeholder);
+
+			if (text !== "_;") {
```

Comment:
could you instead use a RegExp validation? Because even `_ ;`, `_\t;`, `_\n;` are all valid (although bad style) but will be flagged if we do an exact match for `_;`
The regexp needs to account for all whitespace between `_` and `;`
\+ add tests^